app/controllers/assessment360_controller.rb

```diff
@@ -59,21 +57,27 @@ def all_students_all_reviews
                                                    @meta_review_info_per_stu)
       end
       # calculate average grade for each student on all assignments in this course
-      if @teammate_review_info_per_stu[1] > 0
-        temp_avg_grade = @teammate_review_info_per_stu[0] * 1.0 / @teammate_review_info_per_stu[1]
-        @teammate_review[cp.id][:avg_grade_for_assgt] = temp_avg_grade.round.to_s + '%'
-      end
-      if @meta_review_info_per_stu[1] > 0
-        temp_avg_grade = @meta_review_info_per_stu[0] * 1.0 / @meta_review_info_per_stu[1]
-        @meta_review[cp.id][:avg_grade_for_assgt] = temp_avg_grade.round.to_s + '%'
-      end
+      review_info_per_student(cp,@teammate_review_info_per_stu, @teammate_review)
```

Comment:
Thank you for making teammate_review, etc., a parameter, but "review_info_per_stu" is still an unacceptable name for a method.  It doesn't give any hint of what it actually refers to.  Moreover, it seems to be an array of at least two elements, and nowhere does it say what these elements are!  This is no way to write code.  If necessary, make review_info_per_stu into a hash instead of an array, but better to find a way to get rid of it entirely, in favor of something whose purpose is apparent from its name.